Makefile

```diff
@@ -194,10 +194,10 @@ ifeq ($(dbbackend),postgres)
 	# Remove a previous postgres instance if it exists.
 	docker rm lnd-postgres --force || echo "Starting new postgres container"
 
-	# Start a fresh postgres instance. Allow a maximum of 200 connections so
+	# Start a fresh postgres instance. Allow a maximum of 500 connections so
```

Comment:
Alternatively, we can also limit `sql` package params for lnd as well.

Reply:
Ah this was just me setting it back to the original config. In the end, I've been running tests locally with the existing config plus the parameters suggested by @djkazic above. I did set the default max connections to in LND. Will push cleaned up/latest code tomorrow.